grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -313,6 +320,21 @@ class StockMovementService {
         }
     }
 
+    void createMissingPicklistItem(StockMovementItem stockMovementItem) {
```

Comment:
Is there a way to somehow differentiate this method from this one https://github.com/openboxes/openboxes/pull/894/files#diff-e533ed1bcd344039497bdc95032d4469R338?
The huge similarity in their names and params can be missleading